bezier-rs/docs/interactive-docs/wasm/src/lib.rs

```diff
@@ -9,12 +9,22 @@ struct Point {
 	y: f64,
 }
 
+/// Wrapper of the `Bezier` struct to be used in JS.
 #[wasm_bindgen]
-/// Wrapper of the `Bezier` struct to be used in JS
+#[derive(Clone)]
 pub struct WasmBezier {
```

Comment:
Have you considered using tuple syntax here instead of key-value pair syntax? `pub struct WasmBezier(Bezier);`

Reply:
I made the change to tuple syntax. To support this, I've swapped all instances of `self.internal` to `self.0`. Is the best way of referencing the underlying Bezier object?

Reply:
Yes, unfortunately there's no other way (besides destructuring into a variable assignment on a separate line, which is unnecessarily verbose in this case) besides the `.0` syntax in Rust. That is idiomatic.